source/Coop.IntegrationTests/Clans/ClanTests.cs

```diff
@@ -0,0 +1,299 @@
+﻿using Coop.IntegrationTests.Environment;
+using Coop.IntegrationTests.Environment.Instance;
+using GameInterface.Services.Clans.Messages;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Coop.IntegrationTests.Clans
+{
+    public class ClanTests
+    {
+        internal TestEnvironment TestEnvironment { get; }
+
+        public ClanTests()
+        {
+            // Creates a test environment with 1 server and 2 clients by default
+            TestEnvironment = new TestEnvironment();
+        }
+
+        /// <summary>
+        /// Verify sending ClanNameChange on one client
+        /// Triggers ClanNameChanged on all other clients
+        /// </summary>
+        [Fact]
+        public void ClanNameChange_Publishes_AllClients()
+        {
+            // Arrange
+            var clanId = "TestClan";
+            var clanName = "RealName";
+            var informalName = "FakeName";
+
+            var message = new ClanNameChanged(clanId, clanName, informalName);
+
+            var client1 = TestEnvironment.Clients.First();
+
+            var server = TestEnvironment.Server;
+
+            // Act
+            client1.ReceiveMessage(this, message);
+            server.ReceiveMessage(this, message);
```

Comment:
This should be it's own test. Do the same thing as the client but on the server in this case. Please split server testing into separate tests.